### ISSUE TITLE
not valid for use in process: mapped file has no Team ID and is not a platform binary (signed with custom identity or adhoc?))

### ISSUE THREAD

**Issue by bharatiadkar** [phpredis/phpredis#1790]
### I'm seeing this behaviour on
- OS: MacOS Catalina 10.15.5
- Redis:
- PHP:7.3.11
- phpredis:

### I've checked
- [ ] There is no similar issue from other users
- [ ] Issue isn't fixed in `develop` branch

After reading all the blogs and post out there I was able to build redis.so .

But when I try to add 

extension = /usr/local/php/extensions/redis.so
I get  the error
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/php/extensions/redis.so' (tried: /usr/local/php/extensions/redis.so (dlopen(/usr/local/php/extensions/redis.so, 0x0009): code signature in (/usr/local/php/extensions/redis.so) not valid for use in process: mapped file has no Team ID and is not a platform binary (signed with custom identity or adhoc?)), /usr/lib/php/extensions/no-debug-non-zts-20180731//usr/local/php/extensions/redis.so.so (dlopen(/usr/lib/php/extensions/no-debug-non-zts-20180731//usr/local/php/extensions/redis.so.so, 0x0009): dlopen(): file not found: /usr/lib/php/extensions/no-debug-non-zts-20180731//usr/local/php/extensions/redis.so.so)) in Unknown on line 0
PHP 7.3.11 (cli) (built: Apr 17 2020 19:14:14) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.11, Copyright (c) 1998-2018 Zend Technologies

Any help will be appreciated.

Thanks,

**Comment by michael-grunder** [comment]
I think this is a duplicate of #1406.  There's some good information in that thread that may be of use.

TL;DR:  This is an issue involving OSX's new security functionality but none of the maintainers use a mac so we can't be super helpful.
